src/encoded/static/components/cart/facet.js

```diff
@@ -50,8 +49,8 @@ const analysisSorter = (facetTerms, analyses) => (
  *
  * @return {string} Combined query string selecting file.analyses titles.
  */
-const analysisFieldMap = (analysisTitles) => (
-    analysisTitles.map((title) => `files.analyses.title=${encodedURIComponent(title)}`).join('&')
+const analysisFieldMap = () => (
```

Comment:
Why. does this function just return the same string always?

Reply:
In this case it's a static string. In a different case it could be different.